_posts/2023-12-13-gift-guide.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: blog_post
+title: "Compiler's 2023 Holiday Gift Guide"
+subtitle: "Searching for that perfect gift for the public service enthusiast in your life? Compiler's got you covered! Here is this year's round-up of our favorite transit-related gift ideas.
+*(P.S. If you didn't make the holiday cut off this year, we think these make great gifts year-round!)*"
```

Comment:
```suggestion
*(P.S. If you didn’t make the holiday cut off this year, we think these make great gifts year-round!)*"
```

Reply:
@machikoyasuda I was hoping to make the P.S. portion of this italicized and possibly in a new paragraph. Is there a way to do that? I tried the *'s and put it on a new line but doesn't seem to be working (CC @angela-tran)

Reply:
```suggestion
<br><br><p><em>P.S. If you didn’t make the holiday cut off this year, we think these make great gifts year-round!</em></p>
```

Reply:
Try this ☝️